langlib/lang.stream/src/main/ballerina/Ballerina.toml

```diff
@@ -1,3 +1,3 @@
 [project]
 org-name = "ballerina"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
This has to be "0.4.0" right? Spec minor version + 0.1.0 for preview features as @hasithaa mentioned?

Reply:
I wasn't aware of that. Will update